samples/tab-personal-quickstart/js/package.json

```diff
@@ -1,22 +1,27 @@
 {
   "name": "microsoft-teams-ext",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "msteams": {
     "teamsAppId": null
   },
   "private": true,
-  "dependencies": { 
-    "@fluentui/react-components": "^9.0.0-rc.10",
+  "dependencies": {
+    "@fluentui/react-components": "^9.15.1",
     "@microsoft/teams-js": "^2.0.0",
+    "proxyquire": "^2.1.3",
```

Comment:
Are we using this?

Reply:
ah sorry early implementation used it but not any more. Will remove